app/src/main/res/layout/fragment_review_contest.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="@dimen/activity_vertical_margin"
+    >
+
+    <TextView
+        android:id="@+id/contest_title_hint"
+        style="@style/AppTheme.TextViewLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/name"
+        android:textColor="@color/colorLightGray"
+        />
+
+    <TextView
+        android:id="@+id/contest_title_button"
+        style="@style/AppTheme.TextViewLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        />
+
+    <View
```

Comment:
Once PR #28 (`gabi/con-140-overview`) is merged you can use `AppTheme.HorizontalRow` style for this